Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "./README.md"
 repository = "https://github.com/gimli-rs/gimli"
 
 [dependencies]
-fallible-iterator = { version = "0.2.0", default-features = false, optional = true }
+fallible-iterator = { version = "0.3.0", default-features = false, optional = true }
```

Comment:
Do you need something from this version? This is part of the public API, so I prefer to leave this until we have other breaking changes.

Reply:
At one point I needed https://github.com/sfackler/rust-fallible-iterator/pull/34, but it seems I don't anymore. Removed this bump.